### ISSUE TITLE
Missing link

### ISSUE THREAD

**Issue by jkiddo** [ehealthsuisse/EPR-by-example#29]
There is no link to any reference environment on https://github.com/ehealthsuisse/EPR-by-example/blob/main/files/gazelle.md

**Comment by msmock** [comment]
Yes. It is stored as a variable to be displayed in the rendered html page https://ehealthsuisse.github.io/EPR-by-example/.
